### PR TITLE
css-layout-classes by jaden bertino

### PR DESCRIPTION
**Pull request by jadenbertino**
image

### CODE REVIEW

File: css-layout-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>css-layout-classes</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="col">
```
Comment (scrumptiousdev) [inline css-layout-classes/index.html:13]: Do we have any styling for `.col`?

File: css-layout-classes/styles.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  display: flex;
+  margin: 24px 0;
+}
+
+.col-one-third {
+  flex-basis: calc(100%/3);
+}
+
+.col-one-fourth {
+  flex-basis: calc(25%);
```
Comment (scrumptiousdev) [inline css-layout-classes/styles.css]: If it's not doing any calculation, we don't have to use `calc`

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]
Superb! Everything looks good! I am just being picky :)

**Comment by jadenbertino** [comment]
Wasn't completely sure what to put for the `.col` class, but here's what I put:
image
To be honest I'm not sure if it's doing anything compared to the default `flex-basis: auto`

**Comment by scrumptiousdev** [comment]
> Wasn't completely sure what to put for the `.col` class, but here's what I put: image To be honest I'm not sure if it's doing anything compared to the default `flex-basis: auto`

Since we are constructing a layout each row or its col(s) should expand 100% — even though it's just one column. So having said that, `.col` can be either `flex-basis: 100%` or `width: 100%`. There is one more property you can use, which is `flex: 1`, which would also make it expand. If you want to explore more on `flex`, here is the example I made:

https://jsbin.com/lihifehiya/1/edit?html,css,output

**Review by scrumptiousdev — APPROVED** [review]
Approved!

**Pull request merged**